apps/src/code-studio/pd/workshop_dashboard/components/survey_results/text_responses.jsx

```diff
@@ -1,8 +1,8 @@
+import he from 'he';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 import {Well} from 'react-bootstrap'; // eslint-disable-line no-restricted-imports
-import _ from 'lodash';
-import he from 'he';
```

Comment:
Total aside, but: looks like we have a worthy challenger to the Go programming language in the "least-searchable name of all time" competition